releases/cloud/point-2024-12-17-release-note.md

```diff
@@ -0,0 +1,36 @@
+---
+description: This article lists improvements and bug fixes in the Syskit Point Cloud version 2024.6.74.1
+---
+
+# December 17, 2024
+
+[Start a free trial](https://www.syskit.com/products/point/free-trial/) and [tell us what you think](https://www.syskit.com/company/contact-us/).
+
+## About Syskit Point Cloud
+
+* **New to Syskit Point Cloud?** Explore Syskit Point Cloud with a 21-day [free trial](https://www.syskit.com/products/point/free-trial/) for an easy and effective way to manage and secure your environment.
+
+* **Already using Syskit Point Cloud?** Syskit Point Cloud is automatically upgraded to the latest version when available. The automatic update occurs outside working hours to ensure minimal interference with your day-to-day business. The new version will start rolling out with this announcement and is expected to reach all customers in the upcoming days.
+
+
+## Improvements & Bug Fixes
+
+* **In the Syskit Point app for Microsoft Teams**, the Requests screen now has default sorting. 
+  * **The “Requested On” column** is automatically sorted in descending order, displaying the most recent requests at the top.
+
+* **When Resolving Access Requests** and completing the Add Users action, the change is immediately applied, and after refreshing the workspace, the added users can be found in the members list. 
```

Comment:
dodao bi tu info da prije to nije radilo dobro za exchange grupe (mail-enabled security groups & distribution lists)